src/wallet/coinselection.cpp

```diff
@@ -74,7 +74,7 @@ struct {
 static const size_t TOTAL_TRIES = 100000;
 
 util::Result<SelectionResult> SelectCoinsBnB(std::vector<OutputGroup>& utxo_pool, const CAmount& selection_target, const CAmount& cost_of_change,
-                                             int max_weight)
+                                             int max_input_weight)
```

Comment:
Ah, yeah `max_input_weight` is a much better name. Thanks!

Reply:
max_selection_weight is also fine I think, both are better than status quo